src/Aws/Sqs/Resources/sqs-2012-11-05.php

```diff
@@ -98,6 +98,11 @@
                     'type' => 'string',
                     'location' => 'aws.query',
                 ),
+                'MessageGroupId' => array(
+                    'required' => true,
```

Comment:
Reference latest SQS model: https://github.com/aws/aws-sdk-php/blob/master/src/data/sqs/2012-11-05/api-2.json#L266, I didn't see `MessageGroupId` to be a required parameter?